zipkin-lens/src/components/DependenciesPage/DependenciesGraph.jsx

```diff
@@ -0,0 +1,172 @@
+    ...base,
+    width: '15rem',
+  }),
+};
+
+const vizStyle = {
+  colorText: theme.palette.primary.contrastText,
+  colorTextDisabled: theme.palette.primary.contrastText,
+  colorConnectionLine: theme.palette.grey[800],
+  colorTraffic: {
+    normal: theme.palette.primary.dark,
+    warning: theme.palette.secondary.dark,
+    danger: theme.palette.secondary.dark,
+  },
+  colorDonutInternalColor: theme.palette.primary.light,
+  colorDonutInternalColorHighlighted: theme.palette.primary.light,
+  colorLabelBorder: theme.palette.primary.dark,
+  colorLabelText: theme.palette.primary.contrastText,
+  colorTrafficHighlighted: {
+    normal: theme.palette.primary.main,
+  },
+};
+
+const propTypes = {
+  selectedNodeName: PropTypes.string,
+  onNodeClick: PropTypes.func.isRequired,
+  edges: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  nodes: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+};
+
+const defaultProps = {
+  selectedNodeName: '',
+};
+
+const DependenciesGraph = React.memo(({
+  selectedNodeName,
+  onNodeClick,
+  edges,
+  nodes,
+}) => {
+  const classes = useStyles();
+  const [filter, selectFilter] = useReducer((_, selected) => (selected ? selected.value : ''), '');
+
+  // A hack that forces to reload vizceral graph...
+  const [reloading, setReloading] = useState(false);
+  useEffect(() => { setReloading(true); }, [filter]);
+  useEffect(() => { setReloading(false); }, [reloading]);
+
+  const shownConnections = useMemo(() => {
+    if (filter) {
+      return edges.filter(edge => edge.source === filter || edge.target === filter);
+    }
+    return edges;
+  }, [edges, filter]);
+
+  const shownNodes = useMemo(() => {
+    if (filter) {
+      return nodes.filter(
+        node => shownConnections.find(
+          edge => edge.source === node.name || edge.target === node.name,
+        ),
+      );
+    }
+    return nodes;
+  }, [filter, nodes, shownConnections]);
+
+  const handleObjectHighlight = useCallback((highlightedObject) => {
+    if (!highlightedObject) {
+      onNodeClick(null);
+      return;
+    }
+    if (highlightedObject.type === 'node' && highlightedObject.getName() !== selectedNodeName) {
+      onNodeClick(highlightedObject.getName());
+    }
+  }, [onNodeClick, selectedNodeName]);
+
+  const maxVolume = useMemo(() => {
+    if (edges.length > 0) {
+      return edges.map(edge => edge.metrics.normal + edge.metrics.danger)
+        .reduce((a, b) => Math.max(a, b));
+    }
+    return 0;
+  }, [edges]);
+
+  const filterOptions = useMemo(() => nodes.map(node => ({
+    value: node.name,
+    label: node.name,
+  })), [nodes]);
+
+  if (reloading) {
+    return null; // Now reloading Vizceral...
+  }
+
+  return (
+    <Box className={classes.root}>
+      <VizceralExt
+        allowDraggingOfNodes
+        targetFramerate={30}
+        traffic={{
+          renderer: 'region',
+          layout: 'ltrTree',
+          name: 'dependencies-graph',
+          updated: new Date().getTime(),
```

Comment:
Looks like vizceral-react uses current time if it's not passed so we shouldn't need to pass anything.